specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -3555,7 +3555,7 @@
           "type": "string"
         },
         "alternativeNames": {
-          "description": "altenative names",
+          "description": "alterative names",
```

Comment:
alternative?

Reply:
Yeah, that makes more sense. Updated and rebased